doc/item_bonus.txt

```diff
@@ -198,7 +198,14 @@ bonus2 bSkillUseSPrate,sk,n;		Decreases SP consumption of skill sk by n%
 Atk/Def
 -------
 bonus2 bSkillAtk,sk,n;    		Increases damage of skill sk by n%
-bonus bSkillRatio,n;			Adds n to the skillratio of all attacks/skills that use it
+bonus bSkillRatio,n;			Adds n to the skillratio of all attacks/skills
+					Detailed Explanation (all examples assume n=5):
+					- Normal attacks will have their skillratio increased (100% -> 105%)
+					- Only skills that have a skill ratio are affected, skills that deal fixed damage are not affected
+					- Both physical and magical skills are affected
+					- Bonus benefits skills with low skill ratio more (200% -> 205%, 800% -> 805%)
+					- Bonus is applied to each hit of skills that multiply damage by number of hits (100%*8 -> 105%*8)
```

Comment:
These 2 are more like example, they could be removed

Reply:
Isn't it necessary to understand it?

Reply:
I don't think so, but let's see what others think

Reply:
I think the explanation is too long.
All others only have half a line, without any examples.